google/gapic/longrunning/operations_client.py

```diff
@@ -0,0 +1,279 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
We need to switch this to BSD to conform to the rest of GAX. This is supposed to be [configured in the GAPIC config here](https://github.com/googleapis/googleapis/blob/master/google/longrunning/longrunning_gapic.yaml#L17), but I expect it's not working because Python isn't on MVVM and the license is hardcoded into the template files.

In a separate toolkit PR, could you update the Python templates to select the correct license based on that GAPIC config field?